eq-author/src/components/CommentsPanel/index.test.js

```diff
@@ -121,26 +206,36 @@ describe("Comments Pane", () => {
     };
   });
 
+  const renderWithContext = (component, ...props) =>
+    render(
+      <MeContext.Provider value={{ me: user }}>{component}</MeContext.Provider>,
+      ...props
+    );
+
   it("should render comments txt area", async () => {
-    const { getByTestId } = render(<CommentsPanel />, { ...props });
+    window.HTMLElement.prototype.scrollIntoView = function() {};
```

Comment:
If we need to mock it out can we do this in a before all and then reset it at the end. Doesn't matter too much but it just makes it a little more obvious its being mocked.

Also could we do `window.HTMLElement.prototype.scrollIntoView = jest.fn();`